.github/workflows/build.yml

```diff
@@ -16,7 +16,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Format Rust code
+      - name: Install nightly rustfmt
+        run: |
+          rustup default nightly
```

Comment:
Instead of changing the default, you can use `cargo +nightly fmt ...` in the next step. I think there's a similar thing for `rustup` to add a component to a specific version.

Reply:
Sure, one can do that... :slightly_smiling_face:  However, I don't like how the `+foo` deviates from the normal command line syntax.